azimuth_capi/models/v1alpha1/cluster.py

```diff
@@ -156,6 +156,10 @@ class ClusterSpec(schema.BaseModel):
         default_factory = AddonsSpec,
         description = "Describes the optional addons that should be enabled for the cluster."
     )
+    createdByUsername: schema.Optional[str] = None
+    createdByUserId: schema.Optional[str] = None
+    updatedByUsername: schema.Optional[str] = None
+    updatedByUserId: schema.Optional[str] = None
```

Comment:
Can you use `snake_case` like all the other fields please? The `kube-custom-resource` package handles the conversion to and from the `pascalCase` versions given in the Kubernetes CRDs and snake case is more Pythonic (and, IMHO, easier to read).

Reply:
I keep forgetting it did that, as I didn't know when I created the caas operator and cut and paste this all from there.